Lecture7/Testing_Task1/Helper/WebDriverUtils/WebDriverUtils.java

```diff
@@ -0,0 +1,18 @@
+package Helper.WebDriverUtils;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import static Helper.WebDriverUtils.ChromeDriverSettings.getDriverName;
+import static Helper.WebDriverUtils.ChromeDriverSettings.getLocation;
+
+public class WebDriverUtils {
+
+    private WebDriverUtils() {
+    }
+
+    public static WebDriver useChromeWebDriver() {
+        System.setProperty(getDriverName(), getLocation());
```

Comment:
constants or set properties in @BeforeClass . Test

Reply:
fixed